consumer/guides/mraa/install.md

```diff
@@ -21,40 +31,262 @@ A few useful commands will help us make sure everything on the board is current:
 **Commands:**
 
 ```shell
-$ sudo apt-get update
-
-$ sudo apt-get upgrade
-
-$ sudo apt-get dist-upgrade
+$ sudo apt update && \
+  sudo apt upgrade -y && \
+  sudo apt dist-upgrade -u -y && \
+  sudo apt-get clean && \
+  sudo apt autoremove -y
 ```
 
-> Note: If at any point during this process you are prompted with a 'Y/N', select Y and press Enter.
-
 ## Install as a package
 
 The following command installs MRAA library as a package.
+
 ```shell
-$ sudo apt-get install libmraa-dev
+$ sudo apt-get install -y libmraa-dev
 ```
 
 ## Build from source
 
-Following instructions specifies installing MRAA library from source.
+Following sections specifies installing MRAA library from source. We build and install dependencies first.
+If you want to use legacy nodejs you can skip nodejs and SWIG sections and just get them from their packages.
+If you want to be current, than do alls teps as outlined below.
+
+**Install extra tool packages**
+
+```shell
+sudo apt-get install -y arduino-mk arduino git build-essential cmake bison doxygen graphviz python-sphinx python3-sphinx libudev-dev libgtest-dev libjson-c-dev autoconf libtool python-dev pkg-config libpcre3-dev
+sudo apt-get clean
+sudo apt autoremove -y
+```
+
+**We install all repos into the git directory to stay organized**
```

Comment:
These informations are very useful to someone who likes to use JAVA bindings but not for everyone. Can you please create two sections? One for a newcomer and another for advanced user? If you don't have time, no issues, I'll do it once this guide gets merged.